pkg/network/tracer/tracer_test.go

```diff
@@ -773,40 +780,46 @@ func TestUDPSendAndReceive(t *testing.T) {
 	require.NoError(t, err)
 	defer tr.Stop()
 
-	cmd := exec.Command("../testdata/simulate_udp.sh")
-	if err := cmd.Run(); err != nil {
-		require.NoError(t, err)
-	}
+	server := NewUDPServerOnAddress("127.0.0.1:8001", func(buf []byte, n int) []byte {
```

Comment:
Does `:0` work  here ?

Reply:
I tried that. Apparently the port does not get bound and so the connection to the server shows up as outgoing, not incoming.

Reply:
Did you see that `bind` wasn't called (via `BPFDebug` or `strace`)? I wonder if `bind` was indeed called, but our code didn't extract the port/address correctly. I just merged the change to use `inet_bind` and `inet6_bind`, so it might be worth trying again with that code merged into this branch.

Reply:
Oh, you're right. The port was showing up as `0` in the kprobe so our code was ignoring it.

Reply:
Its still the same result:
```
           <...>-483148 [001] .... 1210141.822463: 0: ERR(sys_enter_bind): sin_port is 0
```